vizro-core/docs/pages/user_guides/tabs.md

```diff
@@ -0,0 +1,121 @@
+                                        x="continent",
+                                        y="lifeExp",
+                                        color="continent",
+                                    ),
+                                ),
+                            ],
+                        ),
+                        vm.Container(
+                            title="Tab II",
+                            components=[
+                                vm.Graph(
+                                    figure=px.scatter(
+                                        gapminder_2007,
+                                        title="Graph 3",
+                                        x="gdpPercap",
+                                        y="lifeExp",
+                                        size="pop",
+                                        color="continent",
+                                    ),
+                                ),
+                            ],
+                        ),
+                    ],
+                ),
+            ],
+            controls=[vm.Filter(column="continent")],
+        )
+
+        dashboard = vm.Dashboard(pages=[page])
+
+        Vizro().build(dashboard).run()
+        ```
+    === "app.yaml"
+        ```yaml
+        # Still requires a .py to register data connector in Data Manager and parse yaml configuration
+        # See from_yaml example
+        pages:
+          - components:
+              - tabs:
+                  - title: Tab I
+                    type: container
+                    components:
+                      - type: graph
+                        figure:
+                            _target_: bar
+                            data_frame: gapminder_2007
+                            title: Graph 1
+                            x: continent
+                            y: lifeExp
+                            color: continent
+                      - type: graph
+                        figure:
+                            _target_: box
+                            data_frame: gapminder_2007
+                            title: Graph 2
+                            x: continent
+                            y: lifeExp
+                            color: continent
+                  - title: Tab II
+                    type: container
+                    components:
+                      - type: graph
+                        figure:
+                            _target_: scatter
+                            data_frame: gapminder_2007
+                            title: Graph 3
+                            x: gdpPercap
+                            y: lifeExp
+                            size: pop
+                            color: continent
+              type: tabs
+          controls:
+              - type: filter
+                column: continent
+          title: Tabs
+        ```
+    === "Result"
+       [![Tabs]][Tabs]
+
+    [Tabs]: ../../assets/user_guides/components/tabs.png
```

Comment:
The example currently only includes **single tabs with two containers**.

Do we want to add:
- Two tabs with containers on one page?
- Single Tabs with nested containers (this one is relevant if we want to show how to add a different title to the container)?
-

Reply:
**Two tabs with container on one page:**
![Screenshot 2024-01-24 at 10 08 31](https://github.com/mckinsey/vizro/assets/90609403/8b56a837-03e0-4154-9278-ceec4cc71c1d)

Reply:
**Single tabs with nested containers:**
- Would remove the container below and only show the single tabs with two tab secitons

![Screenshot 2024-01-24 at 10 08 50](https://github.com/mckinsey/vizro/assets/90609403/f4dd3d7d-75b0-48c9-a342-5e9487d58378)

Reply:
I would keep it even leaning by not adding most of those notes at all 😅 

* Multiple tabs on one page? ➡️ this could be a nice example but also fine without
* Single Tabs with nested containers (this one is relevant if we want to show how to add a different title to the container)? ➡️ I would just ignore this altogether for now